bedrock-codec/src/main/java/org/cloudburstmc/protocol/bedrock/data/EncodingSettings.java

```diff
@@ -50,7 +50,7 @@ public class EncodingSettings {
 
     /**
      * A {@link EncodingSettings} instance for implementations
-     * that don't need such limits. (e.g. Proxy server client <-> downstream server connection)
```

Comment:
You need to use HTML sequences.

`&lt;-&gt;`

Reply:
Resolved.